### ISSUE TITLE
How to set Arguement to prevent overfitting?

### ISSUE THREAD

**Issue by tjusxh** [BVLC/caffe#1633]
I set Solver use SGD.the train loss in 0.001 or 0.0001.But the test loss = 10.0+.I think it's over fitting.How to correct the arguments?
Thanks

**Comment by Prasanna1991** [comment]
You can use dropout layer!!

**Comment by sguada** [comment]
All questions about usage, installation, code, and applications should be searched for and asked on the [caffe-users mailing list](https://groups.google.com/forum/#!forum/caffe-users).
